src/pathfinding_service/service.py

```diff
@@ -285,56 +285,87 @@ def create_token_network_for_address(
 
     def handle_message(self, message: SignedMessage):
         if isinstance(message, UpdatePFS):
-            self.on_pfs_update(message)
+            try:
+                self.on_pfs_update(message)
+            except MessageException as x:
+                log.info(
+                    str(x),
+                    chain_id=message.canonical_identifier.chain_identifier,
+                    token_network_address=message.canonical_identifier.token_network_address,
+                    channel_identifier=message.canonical_identifier.channel_identifier,
+                    updating_capacity=message.updating_capacity,
+                    other_capacity=message.updating_capacity,
+                )
         else:
             log.info('Ignoring unknown message type')
 
     def on_pfs_update(self, message: UpdatePFS):
+        token_network_address = to_checksum_address(
+            message.canonical_identifier.token_network_address,
+        )
         log.info(
-            'Received Balance Update',
-            token_network_address=message.token_network_address,
-            channel_identifier=message.channel_identifier,
+            'Received Capacity Update',
+            token_network_address=token_network_address,
+            channel_identifier=message.canonical_identifier.channel_identifier,
         )
 
         assert is_checksum_address(message.updating_participant)
         assert is_checksum_address(message.other_participant)
 
         # check if chain_id matches
-        if message.chain_id != self.chain_id:
-            log.info(
-                'Received balance update with unknown chain id',
-                chain_id=message.chain_id,
-            )
-            return
+        if message.canonical_identifier.chain_identifier != self.chain_id:
+            raise MessageException('Received Capacity Update with unknown chain identifier')
```

Comment:
*with*  or *for*? And should we lowercase *capacity update*?

Reply:
the message shows an unknown chain ID. So I would say received something "with"

Reply:
imho "Capacity Update" should be uppercase since we use it as a term?!